src/StackAdmin/Storage.Management/Commands.Management.Storage/Microsoft.Azure.Commands.Management.Storage.dll-Help.psd1

```diff
@@ -0,0 +1,111 @@
+#
+# Module manifest for module 'Microsoft.Azure.Commands.Management.Storage'
+#
+# Generated by: Microsoft Corporation
+#
+# Generated on: 04/01/2015
+#
+
+@{
+
```

Comment:
this one got deleted in preview, you are adding it back

Reply:
Done.